SECURITY.md

```diff
@@ -12,3 +13,62 @@
 Please report any vulnerabilities via our [Bug Bounty Program](https://bugcrowd.com/owaspzap).
 
 Remote Code Execution for this program will be rewarded at __$1000__
+
+## Reporting a Bug
+
+Please report any bugs via our issue [Bug Issue Tracker](https://github.com/zaproxy/zaproxy/issues/new/choose)
+
+## Security Advisories
+
+Currently, there are not any published security advisories.
+
+## Scan Policy
+
+ZAP supports both active and passive scanning rules.
+
+All rules are contained in add-ons so that they can be updated quickly and easily.
+
+Active scanning is an attack on those targets.
+You should NOT use it on web applications that you do not own.
+
+It should be noted that active scanning can only find certain types of vulnerabilities.
+Logical vulnerabilities, such as broken access control, will not be found by any active or automated vulnerability scanning.
+
+Manual penetration testing should always be performed in addition to active scanning to find all types of vulnerabilities.
+
+Please see the Terms of Use and Disclaimer for restrictions.
+
+## Terms of Use
+
+By using Zaproxy, you agree to comply with the terms and conditions below. Terms and conditions may be subject to change or modification at any time. Users are responsible for reviewing the terms regularly to ensure compliance with the latest version.
+
+1. Authorized Usage: You may only use Zaproxy for the purpose of assessing the security of web applications for which you have explicit and authorized access, permission, or ownership. Unauthorized scanning is strictly prohibited and unlawful.
```

Comment:
We never use `Zaproxy` in this context, so this should be `ZAP`.
https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/#style-guidelines